.hintrc

```diff
@@ -1,18 +0,0 @@
-{
-    "connector": {
-      "name": "local",
-      "options": {
-        "pattern": ["**", "!.git/**", "!node_modules/**"]
-      }
-    },
-    "extends": ["development"],
-    "formatters": ["stylish"],
-    "hints": [
-      "button-type",
-      "disown-opener",
-      "html-checker",
-      "meta-charset-utf-8",
-      "meta-viewport",
-      "no-inline-styles:error"
-    ]
```

Comment:
- [ ] As stated by the previous reviewer, kindly add this file content back. It's very possible you have it deleted on the repo, but it's present on your local machine. I've attached a screenshot of files in this branch you're working on to see that it's not present. The linter file is very important to safely check your code and will be a major requirement going forward i future projects. 

-  Kindly add thee file back with its content, and you're good to go👍 

## Branch files:
![image](https://user-images.githubusercontent.com/50990925/227386432-78026b82-c053-4776-9482-38908da41cf3.png)

Reply:
@Shedrack-Sunday added the linters files to Branch, all linters files have to be in both branches or only on the second branch?